SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -3531,6 +3533,136 @@ bool USpatialNetDriver::HasTimedOut(const float Interval, uint64& TimeStamp)
 	return false;
 }
 
+void USpatialNetDriver::ServerReplicateActors_BuildConsiderList(TArray<FNetworkObjectInfo*>& OutConsiderList, const float ServerTickTime)
+{
+	TArray<FNetworkObjectInfo*> TmpConsiderList;
+	TmpConsiderList.Reserve(GetNetworkObjectList().GetActiveObjects().Num());
+
+	Super::ServerReplicateActors_BuildConsiderList(TmpConsiderList, ServerTickTime);
+
+	// Cache if we are running with Spatial Networking to avoid getting the settings in every iteration of the loop.
+	const bool bSpatialNetworking = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
+	if (!bSpatialNetworking)
+	{
+		return;
+	}
+
+	for (FNetworkObjectInfo* const& ActorInfo : TmpConsiderList)
+	{
+		AActor* Actor = ActorInfo->Actor;
+		check(Actor != nullptr);
```

Comment:
Personally I would not assert here, maybe just a if/continue, optionally with an ERROR log (or lesser log)?

Reply:
I've removed the assert as it is not needed. They don't check for it in the base method in NetDriver.